variables/shared.yaml

```diff
@@ -84,3 +84,8 @@ variables:
     value: "vh-infra-core-${{ parameters.env }}-sql-username"
   - name: vhSqlPasswordSecret
     value: "vh-infra-core-${{ parameters.env }}-sql-password"
+
+  - name: dotnetVersion
+    value: '8.x'
+  - name: efDotnetToolVersion
```

Comment:
this doesn't appear to be used, can it be removed? Don't think user API uses EF

Reply:
removed, good point